src/genie/libs/parser/asa/tests/test_show_route.py

```diff
@@ -239,16 +219,15 @@ class test_show_ip_route(unittest.TestCase):
                                         },
                                     },
                                 },
-                                'date': '2w1d',
```

Comment:
Hi @lookbeat ,

Once you put `date` back to the schema, the parser class should parse the value and insert it into the parsed output, but now it seems it is missed.

Reply:
Cool, done. Thanks

```text
(genieparser) tests - asartsv4 $> python -m unittest
............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................./Users/pedr7260/py/genieparser/src/genie/libs/parser/iosxr/show_controllers.py:78: FutureWarning: Possible nested set at position 74
  p2 = re.compile(r'^mac\=(?P<mac>[A-Fa-f0-9:]+) +vlan=(?P<vlan>\d+)'
/Users/pedr7260/py/genieparser/src/genie/libs/parser/iosxr/show_controllers.py:78: FutureWarning: Possible nested set at position 176
  p2 = re.compile(r'^mac\=(?P<mac>[A-Fa-f0-9:]+) +vlan=(?P<vlan>\d+)'
................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................./Users/pedr7260/py/genieparser/src/genie/libs/parser/utils/common.py:604: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  cmd_node = root.getchildren()[0]
/Users/pedr7260/py/genieparser/src/genie/libs/parser/utils/common.py:617: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  cmd_node = cmd_node.getchildren()
....................................................................../Users/pedr7260/py/genieparser/src/genie/libs/parser/nxos/show_bgp.py:8976: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  for item in nei.getchildren():
/Users/pedr7260/py/genieparser/src/genie/libs/parser/nxos/show_bgp.py:8984: DeprecationWarning: This method will be removed in future versions.  Use 'list(elem)' or iteration over elem instead.
  root = item.getchildren()[0]
............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
----------------------------------------------------------------------
Ran 3872 tests in 21.794s

OK
(genieparser) tests - asartsv4 $>
```